target/linux/ath79/dts/ar9341_tplink_tl-wr841-v8.dts

```diff
@@ -0,0 +1,173 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "ar9341.dtsi"
+
+/ {
+	model = "TP-Link TL-WR841N/ND v8";
+	compatible = "tplink,tl-wr841-v8";
```

Comment:
Missing soc compatible string. It should be
compatible = "tplink,tl-wr841-v8", "qca,ar9341";